lua/wire/client/pickcolor_browser.lua

```diff
@@ -0,0 +1,109 @@
+--[[
```

Comment:
In its current form, I believe it only works for E2 since it looks like it outputs the colors in "vec" or "vec4" form. See my suggestions